discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala

```diff
@@ -79,6 +79,8 @@ class MarathonApiSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
 
     val request = HttpRequest(uri = uri)
 
+    system.log.info("Requesting seed nodes by: {}", request.uri)
```

Comment:
Good however please using a logger made like this:

`val log = Logging(getClass, system)` since we should use the "right logger"

Reply:
Fixed. Thanks, @ktoso.